### ISSUE TITLE
Provide a Timmy::VERSION constant

### ISSUE THREAD

**Issue by drzraf** [mindkomm/timmy#48]
So that users can anticipate library's evolution and handle breaking changes smoothly.

**Comment by gchtr** [comment]
Good idea! Added in https://github.com/mindkomm/timmy/releases/tag/0.14.9 and https://github.com/mindkomm/timmy/releases/tag/1.0.0-beta.2.
